docs/.link-check-errors.txt

```diff
@@ -1,4 +1,5 @@
 Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
+- https://mui.com/joy-ui/customization/theme-colors/#default-color-tokens/
```

Comment:
This should not happen.

Reply:
Yeah, no clue why the tests were not happy with this on this PR... just ran the commands they were asking 🤷‍♂️

Reply:
Ah, it was because `joy-ui/customization/theme-colors/#default-color-tokens/` before. Notice the `/` at the end.